app/pages/news/index.vue

```diff
@@ -57,54 +56,70 @@ export default {
     }
   },
   computed: {
-    lastPage () {
-      return Math.round(this.numberOfTotal / this.numberOfItems - 1)
+    fetchCount () {
+      return this.newsFilteredList.length + this.numberOfItems
     }
   },
   async mounted () {
-    this.newsFilteredList = await this.$axios.get(`${newsBaseURL}&per_page=${this.numberOfItems}&offset=0`)
-      .then(res => res.data)
-    this.numberOfTotal = await this.$axios.get(`${newsBaseURL}&per_page=${this.numberOfItems}`)
-      .then(res => parseInt(res.headers['x-wp-total']))
+    this.newsFilteredList = await this.useFetch()
+    this.numberOfTotal = await this.useFetch({ type: 'totalNumber' })
   },
```

Comment:
Since we are struggling with memory leak issues, maybe the safest is to nuke out the results from memory, sometinkg like this
```
  beforeDestroy () {
    this.newsFilteredList = null
  },
```